android/quest/src/main/java/org/smartregister/fhircore/quest/ui/report/measure/MeasureReportViewModel.kt

```diff
@@ -250,16 +253,20 @@ constructor(
   ) {
     val measureReport =
       withContext(dispatcherProvider.io()) {
-        fhirOperator.evaluateMeasure(
-          measureUrl = measureUrl,
-          start = startDateFormatted,
-          end = endDateFormatted,
-          reportType = POPULATION,
-          subject = null,
-          practitioner = practitionerDetails?.id,
-          lastReceivedOn = null // Non-null value not supported yet
-        )
-      }
+          kotlin.runCatching {
+            fhirOperator.evaluateMeasure(
+              measureUrl = measureUrl,
+              start = startDateFormatted,
+              end = endDateFormatted,
+              reportType = POPULATION,
+              subject = null,
+              practitioner = null /*practitionerDetails?.id*/,
```

Comment:
what's w/comment here? we should use // if we still a comment here and add some text to explain

Reply:
Added TODO and ticket

Reply:
Looks like it is fixed in latest main. Fixed